exla/lib/exla/computation.ex

```diff
@@ -70,6 +72,20 @@ defmodule EXLA.Computation do
     }
   end
 
+  def compile(
+        %EXLA.MLIR.Function{module: module, xla_return_shape: ret_shape},
+        client,
+        arg_shapes,
+        _opts
+      ) do
+    EXLA.MLIR.Module.compile(
+      module,
+      client,
+      arg_shapes,
+      EXLA.Shape.make_tuple_shape([ret_shape])
```

Comment:
@josevalim we need to think about how to deal with this. EXLA wraps the result in a tuple automatically, so I just hardcoded this output value.

Reply:
Is there an issue in always making it a tuple? A single-element tuple has no overhead afaik and it helped address bugs between CPU and GPU.

Reply:
Not exactly an issue. The problem right now is that ret_shape was taken from a tensor-return. I assume that returning a tuple or another container would need something else for actually handling here.